javascriptv3/example_code/iam/src/iam_getrole.js

```diff
@@ -0,0 +1,36 @@
+/* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+ABOUT THIS NODE.JS EXAMPLE: This example works with AWS SDK for JavaScript version 3 (v3),
+which is available at https://github.com/aws/aws-sdk-js-v3. This example is in the 'AWS SDK for JavaScript v3 Developer Guide' at
+https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/iam-examples-managing-users.html.
+
+Purpose:
+iam_getrole.js demonstrates how to retrieve information about the specified role, including the role's path, GUID, ARN, and the role's trust policy that grants permission to assume the role.
```

Comment:
iam_getrole.js demonstrates how to retrieve information about the specified role, including the role path, GUID, ARN, and the trust policy that grants permission to assume the role.

Reply:
done